test/functional/foreman_bootdisk/api/v2/disks_controller_test.rb

```diff
@@ -29,5 +29,14 @@ class ForemanBootdisk::Api::V2::DisksControllerTest < ActionController::TestCase
       assert_response :success
       assert_equal "ISO image", @response.body
     end
+
+    test "should generate subnet host image" do
+      ForemanBootdisk::ISOGenerator.expects(:generate).with(has_entry(:ipxe => regexp_matches(/host template/))).yields("temp ISO")
```

Comment:
Why's this matching host template, is it using the host-specific rather than generic template?  I'm not sure that I follow.